juniper_official/Solutions/Krt-async-stuck/krtasync.playbook

```diff
@@ -0,0 +1,157 @@
+/*
```

Comment:
481
Entering configuration mode
The configuration has been changed but not committed
|/rules/juniper_official/Solutions/Krt-async-stuck/krtasync.playbook:155:(8) syntax error: playbook
/rules/juniper_official/Solutions/Krt-async-stuck/krtasync.playbook:157:(5) error recovery ignores input until this point: }
load complete (2 errors)